docs/pelion_ready/device-management-config.md

```diff
@@ -84,13 +86,19 @@ To run the tests:
 $ mbed test -t <TOOLCHAIN> -m <BOARD> -n simple*dev*connect --run -v
 ```
 
-## 2. Bootloader configuration
+## Bootloader configuration
 
-After you've successfully passed the "Connect" tests as described above, you can enable firmware update feature by adding a bootloader to your application.
+When the "connect" tests have passed, you can enable the firmware update feature by adding a bootloader to your application.
```

Comment:
Are there any circumstances under which someone wouldn't do this? Should it just be an imperative 'enable' minus 'you can'?